src/OrchardCore.Modules/OrchardCore.ContentTypes/README.md

```diff
@@ -52,10 +52,12 @@ public class Migrations : DataMigration
 
 ### Creating a new Content Type
 
-The following example creates a new Content Type named `Product`.
+The following example creates a new Content Type named `Product` with a `Product` part.
 
 ```csharp
-_contentDefinitionManager.AlterTypeDefinition("Product");
+_contentDefinitionManager.AlterTypeDefinition("Product", type => type
+                .WithPart("Product")
```

Comment:
I don't think it should be there. We add a `Product` part only to add fields to it, so this should be explained only in the fields section.